system/seboolean.py

```diff
@@ -182,6 +182,9 @@ def main():
     result = {}
     result['name'] = name
 
+    if 'selinux_boolean_sub' in selinux:
```

Comment:
``` python
>>> 'selinux_boolean_sub' in selinux
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'module' is not iterable
```

Maybe one of the following would work:
``` python
try:
    name = selinux.selinux_boolean_sub(name)
except AttributeError:
    # selinux_boolean_sub allows sites to rename a boolean and alias the old name
    # Feature only available in selinux library since 2012.
    pass
```
``` python
if hasattr(selinux, 'selinux_boolean_sub'):
    # selinux_boolean_sub allows sites to rename a boolean and alias the old name
    # Feature only available in selinux library since 2012.
    name = selinux.selinux_boolean_sub(name)
```